api/swagger/swagger.yaml

```diff
@@ -192,6 +207,22 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
+    delete:
```

Comment:
delete should be an authentication call and can only be deleted by the author or an admin user